### PR TITLE
Add contributions to PPP

### PR DESCRIPTION
**Pull request by Justinhoejj**

### CODE REVIEW

File: docs/team/justinhoe.md [hunk 1]
```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
...
+Connections is a desktop address book application used for teaching Software Engineering principles. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. 
+It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The display contacted sorted by birthdays in ascending order, starting with the
```
Comment (Jiwei99) [inline docs/team/justinhoe.md:15]: `The display contacted sorted` sounds a bit awkward?

File: docs/team/justinhoe.md [hunk 2]
```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The display contacted sorted by birthdays in ascending order, starting with the 
+      nearest upcoming birthday all the way through the latest birthday and then looping back 
+      to the earliest birthday.
+    * The list is responsive to changes in contact details be it add, delete or edit command. The list
```
Comment (Jiwei99) [inline docs/team/justinhoe.md:18]: `be it **using** add`

File: docs/team/justinhoe.md [hunk 3]
```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
...
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The display contacted sorted by birthdays in ascending order, starting with the 
+      nearest upcoming birthday all the way through the latest birthday and then looping back 
+      to the earliest birthday.
+    * The list is responsive to changes in contact details be it add, delete or edit command. The list 
+      reorders contacts where appropriate to ensure it stays sorted as described above.
+    * Birthday at present day and coming within 7 days are color coded as well as personalised message.
```
Comment (Jiwei99) [inline docs/team/justinhoe.md:20]: `Birthday at present day and upcoming birthdays within 7 days are color coded with personalised message.` would be better?

**Review by Jiwei99 — APPROVED** [review]
Otherwise lgtm

File: docs/team/justinhoe.md (hunk 1 above)
Comment (Justinhoejj) [reply docs/team/justinhoe.md:15]: U are absolutely right sir

File: docs/team/justinhoe.md [hunk 4]
```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
 
 ### Project: Connections
 
-Connections is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Connections is a desktop address book application used for teaching Software Engineering principles.
```
Comment (yucheng11122017) [inline docs/team/justinhoe.md:8]: Btw should be in caps! the above also :)

File: docs/team/justinhoe.md [hunk 5]
```diff
@@ -3,25 +3,50 @@ layout: page
...
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The list displays contacts sorted by birthdays in ascending order, starting with the 
+      nearest upcoming birthday all the way through the latest birthday and then looping back 
+      to the earliest birthday.
+    * The list is responsive to changes in contact details be it using add, delete or edit command. The list 
+      reorders contacts where appropriate to ensure it stays sorted as described above.
+    * Birthday at present day and upcoming birthdays within 7 days are color coded and provided with personalised message.
```
Comment (yucheng11122017) [inline docs/team/justinhoe.md:20]: Maybe mention its different colours? :)

File: docs/team/justinhoe.md [hunk 6]
```diff
@@ -3,25 +3,50 @@ layout: page
...
+    * This feature would allow our target user, a person who loves birthday celebrations, always wish contacts on
```
Comment (yucheng11122017) [inline docs/team/justinhoe.md:21]: Err maybe delete the always absolute terms are not fun HAHA

**Review by yucheng11122017 — APPROVED** [review]
LGTM besides some minor stuff :)

**Pull request merged**